ui/k-info-panel/edit_window.kytos

```diff
@@ -676,6 +725,13 @@
         float: right;
         background: #372C5E;
     }
+    .window-buttons .window-extend-button button {
+        float: right;
+        background: #372C5E;
+    } 
+    .minute-input-field {
```

Comment:
Could you align the `minute-input-field` as float: right? As it is, I found it difficult to locate it. 

If we could have something like:  `k-input | extend button | finish button` aligned to the right side.

Reply:
@rohanCodes001 appreciated these last commits. @rmotitsuki could you add this PR to your review list? Let us know if you also have any advise or suggestions for CSS that you think it's crucial to have here:

![2022-04-18-111256_1634x742_scrot](https://user-images.githubusercontent.com/1010796/163820720-5b85b89a-dbd8-4edc-97f5-076742df09a6.png)